Cell.py

```diff
@@ -1,15 +1,31 @@
 # Cell.py
+from curses.ascii import EM
+from queue import Empty
+
+
 class Cell:
 	def __init__(self, loc):
+		print("Cell class says hi.")
 		# Parameters:
 		# 	- loc: list of 2 integers
 		# 		tells the location of the cells
 		self.loc 				= loc
-		self.isOccupied 		= False
 		self.isTransformable 	= False
 		self.Score 				= 0
 		self.isAvailable		= False
 		self.isBlack			= ((loc[0] + loc[1]) % 2) == 0
-
 		self.piece				= "" # chess piece objects
```

Comment:
You can modify this line to set the initial value of self.Piece equal to None
self.piece				= None

Reply:
I didn't know python has "None" =))) Was trying "Null" or "Empty" and got troubles with them.